pkg/controller/status/status.go

```diff
@@ -347,6 +375,9 @@ func (c *Controller) updateStatus(initial bool) error {
 // OperatorDisabled reports when the primary function of the operator has been disabled.
 const OperatorDisabled configv1.ClusterStatusConditionType = "Disabled"
 
+// Uploading reports true when the operator is successfully uploading
+const Uploading configv1.ClusterStatusConditionType = "Uploading"
```

Comment:
We generally try not to create new ClusterStatusConditions.  Putting it on a new condition means it won't show up in any user interface except the generic ones, which may be ok, but I'm not certain yet of the name.  For now, let's clean up the PR as if the name is fine and then we can discuss naming.